angular.json

```diff
@@ -17,6 +17,7 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
+            "aot": true,
```

Comment:
is this required for the dialog popup? Not sure why the compilation type is changing. It looks like ahead of time compilation gives better load times for the user, but I'm just not sure why it changed or if there was a reason we were using JIT compilation before.

Reply:
@jamesbews do you know why this might have changed? I'm sure its fine. I was just curious.

Reply:
No its not required. It stands for ahead of time compilation. It won't harm anything